public/app/features/alerting/unified/api/sloApi.ts

```diff
@@ -0,0 +1,35 @@
+import { SupportedPlugin } from '../types/pluginBridges';
+
+import { alertingApi } from './alertingApi';
+
+export interface Label {
+  key: string;
+  value: string;
+}
+
+export interface SloAlertingMetadata {
+  annotations?: Label[];
+  labels?: Label[];
+}
+
+export interface SloAlerting {
+  annotations?: Label[];
+  fastBurn?: SloAlertingMetadata;
+  labels?: Label[];
+  slowBurn?: SloAlertingMetadata;
+}
+
+export interface Slo {
+  alerting?: SloAlerting;
+}
+
+const SLO_API_PATH = `/api/plugins/${SupportedPlugin.Slo}/resources/v1`;
+
+export const sloApi = alertingApi.injectEndpoints({
+  endpoints: (build) => ({
+    getSlos: build.query<{ slos: Slo[] }, void>({
+      query: () => ({ url: `${SLO_API_PATH}/slo`, showErrorAlert: false }),
```

Comment:
That's a great point. There's an internal work effort in the Grafana SLO squad to make "auto plugin init" feature a thing, but for now, some stack still needs to manually initiate the SLO app, which means, setup manually service account tokens, since is not defined as user install the plugin. Without such a token, the request is rejected and return the undesired alert error. Since it doesn't break the user flow, it's not a big deal to disable it here, since as soon as the user enters the Grafana SLO app (through IRM Config Tracker) or directly, the user will be asked to set up the tokens. At some point in the future, we'll be able to remove this flag, since "auto plugin init" feature will be a thing for all stacks, but still without an proper ETA from the BE team of Grafana SLO.